src/cljs/athens/views/blocks/core.cljs

```diff
@@ -362,7 +362,7 @@
                 reactions              (and reactions-enabled?
                                             (block-reaction/props->reactions properties))
                 menu                   (r/as-element
-                                         [:> MenuList {:class "anchor"}
```

Comment:
The menu items need to have an `.anchor` class as a parent, otherwise, multiple block selection gets cleared when trying to click one of the buttons. https://github.com/athensresearch/athens/pull/2296